proto/axelar/nexus/v1beta1/query.proto

```diff
@@ -150,4 +150,8 @@ message TransferRateLimit {
   // time_left indicates the time left in the rate limit window
   google.protobuf.Duration time_left = 5
       [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
-};
+}
+
+message MessageRequest { string ID = 1; }
+
+message MessageResponse { exported.v1beta1.GeneralMessage message = 1; }
```

Comment:
```suggestion
message MessageResponse { exported.v1beta1.GeneralMessage message = 1 [ (gogoproto.nullable) = false ]; }
```
to avoid returning null